common/config/rush/pnpm-lock.yaml

```diff
@@ -6948,6 +6952,34 @@ packages:
       - utf-8-validate
     dev: true
 
+  /jest-config/25.4.0:
```

Comment:
Shouldn't this already have been installed?

Reply:
Normally yes, but it seems that the version descriptor is a bit looser for Jest and it chooses to install the 25.5.0 version. Unsure why that is. But I figured I would try to keep the versioning consistent with the one it was released with.